src/interfaces/product.ts

```diff
@@ -0,0 +1,13 @@
+import type Category from './categorie';
+
+interface Product {
+    id: number;
+    title: string;
+    price: number;
+    description: string;
+    images: string[];
+    creationAt: string;
```

Comment:
it's `Date` type. Pls check it and fix for all interfaces where we have Date.

Reply:
fixed